components/accelerator/nvidia/query/nvml/nvml.go

```diff
@@ -88,9 +88,16 @@ type instance struct {
 	// read-only database instance
 	dbRO *sql.DB
 
+	xidEventsStore events_db.Store
```

Comment:
why do we still need this in the nvml pkg if the decision is to only use dmsg for xid detection for now? cc @cardyok

Reply:
Let me remove, as we decided to use dmesg as a source of event

Reply:
removed